test/helpers/setup.js

```diff
@@ -0,0 +1,13 @@
+const chai = require('chai');
+
+module.exports = {
+  should: function () {
+    return require('chai').should();
+  },
+  BigNumber: function (bignumber) {
+    return chai.use(require('chai-bignumber')(bignumber));
+  },
+  shouldWithBigNumber: function (bignumber) {
```

Comment:
I think this may be a bit overkill: I would simply export a single function, and have it return both `should` and `BigNumber`, leaving it up to the user to take what is needed.

```
const BigNumber = web3.BigNumber;
const should = chai.use(require('chai-bignumber')(BigNumber)).should();

module.exports = {
  BigNumber,
  should,
};
```

Reply:
should i remove simple `chai`'s _should_?

Reply:
Not sure what you mean by removing `should`. We need to export it because some tests use the `should` object (e.g. when running `should.Throw(...)`).

Reply:
Not sure what you mean by removing `should`. We need to export it because some tests use the `should` object (e.g. when running `should.Throw(...)` or `should.exist(...)`).

Reply:
Ah, I see what you mean. In your example, I would remove `should`, and rename `shouldForBigNumber` to `should` :)